code/game/objects/items/crayons.dm

```diff
@@ -240,6 +240,11 @@
 	colour = "#FFFFFF"
 	colourName = "white"
 
+/obj/item/toy/crayon/white/chalk
+	name = "detective's chalk"
+	desc = "A stick of white chalk for marking crime scenes."
```

Comment:
Should probably make it plural so it'll be displayed like `that's some detective's chalk.`'
```suggestion
	desc = "A stick of white chalk for marking crime scenes."
	gender = PLURAL
```